pootle/settings/60-translation.conf

```diff
@@ -66,6 +66,35 @@ POOTLE_TM_SERVER = {
         'HOST': 'localhost',
         'PORT': 9200,
         'INDEX_NAME': 'translations',
+        # Name of the Translation Memory.
+        'TM_NAME': 'Local translations',
+        # Weighting factor to ponder prevalence of this Translation Memory in
+        # relation to others. Between 0 and 1.
+        'WEIGHT': 1,
+        # Tells if this is a local Translation Memory that gets automatically
+        # updated every time a user translates, or if it is a Translation
+        # Memory that gets updated using external translation resources by
+        # using the update_external_tmserver management command.
+        'IS_LOCAL': True,
+        # For valid values for MIN_SCORE see
+        # http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/common-options.html#fuzziness
+        'MIN_SCORE': 'AUTO',
+    },
+    'external': {
+        'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
+        'HOST': 'localhost',
+        'PORT': 9200,
+        'INDEX_NAME': 'external-translations',
+        # Name of the Translation Memory.
```

Comment:
Don't repeat comments it only makes for more noise, maintenance burdern and confusion to someone setting this up.  Comment the first instance fully instead.